frameworklauncher/doc/USERMANUAL.md

```diff
@@ -99,16 +99,22 @@ In other words, User do not need to worry about call one API multiple times by d
 * All APIs are DISTRIBUTED THREAD SAFE, to allow multiple distributed Client instances to access.
 In other words, User do not need to worry about call them at the same time in Multiple Threads/Processes/Nodes.
 
+### <a name="BestPractices">Best Practices</a>
+* LauncherService can only handle a finite, limited request volume. User should try to minimize its overall request frequency and payload, so that the LauncherService is not overloaded. To achieve this, User can centralize requests, space out requests, filter respond and so on.
+* Completed Frameworks will ONLY be retained in recent FrameworkCompletedRetainSec, in case Client miss to delete the Framework after FrameworkCompleted. One exclusion is the Framework Launched by DataDeployment, it will be retained until the corresponding FrameworkDescriptionFile deleted in the DataDeployment. To avoid missing the CompletedFrameworkStatus, the polling interval seconds of Client should be less than FrameworkCompletedRetainSec. Check the FrameworkCompletedRetainSec by [GET LauncherStatus](#GET_LauncherStatus).
+
 ### <a name="RootURI">Root URI (LauncherAddress)</a>
```

Comment:
Break long lines.

Reply:
It is one markdown section, break it to two line will cause another section.

---
In reply to: [167463209](https://github.com/Microsoft/pai/pull/320#discussion_r167463209) [](ancestors = 167463209)